scripts/protocol-filters/beefy.ts

```diff
@@ -0,0 +1,91 @@
+import fetch, { RequestInit, Response } from 'node-fetch'
+import { ReferralEvent } from '../types'
+
+export async function filterEvents(
+  events: ReferralEvent[],
+): Promise<ReferralEvent[]> {
+  const filteredEvents = []
+  for (const event of events) {
+    if (await filter(event)) {
+      filteredEvents.push(event)
+    }
+  }
```

Comment:
should this be a `Promise.all` ?  Or does making so many requests at the same time cause issues?

Reply:
With the rate limiting it was faster to have them go one at a time. still pretty fast either way